Dockerfile

```diff
@@ -1,5 +1,36 @@
 # Dockerfile for production builds
-FROM reactioncommerce/node-prod:14.18.1-v1
+# syntax=docker/dockerfile:1.4
+
+FROM node:14.18.1-alpine as deps
```

Comment:
This docker file is better suited for the app folder itself, not In the root of the repo. If you want to build a container image with the root context but use the `apps/reaction/Dockerfile` you can specify this in the `docker build` command like so:
```
docker build -t ... -f ./apps/reaction/Dockerfile .
```